src/rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -144,6 +144,8 @@ PAI_TASK_INDEX=$TASK_INDEX
 PAI_CONTAINER_HOST_IP=$CONTAINER_IP
 PAI_CONTAINER_HOST_PORT_LIST=$CONTAINER_PORTS
 PAI_CONTAINER_ID=$CONTAINER_ID
+ENV
+cat <<'ENV' >> $ENV_LIST
 PAI_DEFAULT_FS_URI={{{ hdfsUri }}}
 PAI_JOB_NAME={{{ jobData.jobName }}}
 PAI_USER_NAME={{{ jobData.userName }}}
```

Comment:
'\n' should also be escaped.
We can construct a user name like 'noname\nENV\nsome-dangerous-commands'